pkg/models/repository_configuration.go

```diff
@@ -9,3 +9,15 @@ type RepositoryConfiguration struct {
 	AccountID string `json:"account_id" gorm:"default:null"`
 	OrgID     string `json:"org_id" gorm:"default:null"`
 }
+
+//When updating a model with gorm, we want to explicitly update any field that is set to
+// empty string.  We always fetch the object and then update it before saving
+// so every update is the full model of user changable fields.
+// So OrgId and account Id or excluded
```

Comment:
haha! Go my thwart my terrible spelling but it will never overcome my horrendous grammar!